.prout.json

```diff
@@ -0,0 +1,12 @@
+{
+    "checkpoints": {
+        "PROD": {
+            "url": "https://interactive.guim.co.uk/2016/05/brexit-companion/embed/embed.html",
+            "overdue": "3H"
```

Comment:
oh I guess the idea is we would be manually testing in the meantime... fair enough